a small addition to the `changed` flag (#5170)
vitest-dev/vitest: docs/guide/cli.md

  Run tests only against changed files. If no value is provided, it will run tests against uncommitted changes (including staged and unstaged).

>   To run tests against changes made in the last commit, you can use `--changed HEAD~1`. You can also pass commit hash or branch name. If you want to compare it with a branch `--changed=origin/develop`.

Reviewer: ```suggestion
  To run tests against changes made in the last commit, you can use `--changed HEAD~1`. You can also pass commit hash (e.g. `--changed 09a9920`) or branch name (e.g. `--changed origin/develop`).
```

>   If paired with the [`forceRerunTriggers`](/config/#forcereruntriggers) config option it will run the whole test suite if at least one of the files listed in the `forceRerunTriggers` list changes.

Reviewer: ```suggestion
  If paired with the [`forceRerunTriggers`](/config/#forcereruntriggers) config option it will run the whole test suite if at least one of the files listed in the `forceRerunTriggers` list changes. By default, changes to the Vitest config file and `package.json` will always rerun the whole suite.
```
